provisioning/device/src/Transports/Amqp/ProvisioningTransportHandlerAmqp.cs

```diff
@@ -95,18 +95,12 @@ internal ProvisioningTransportHandlerAmqp(ProvisioningClientOptions options)
                 cancellationToken.ThrowIfCancellationRequested();
 
                 bool useWebSocket = _settings.Protocol == ProvisioningClientTransportProtocol.WebSocket;
-                var builder = new UriBuilder
```

Comment:
This URI was being picked apart later, so there was never any need to construct it in the first place.